resolwe/process/runtime.py

```diff
@@ -8,6 +9,15 @@
 
 try:
     from plumbum import local as Cmd  # pylint: disable=unused-import
+    # Log plumbum commands to standard output.
+    root = logging.getLogger('plumbum.local')
+    root.setLevel(logging.DEBUG)
+
+    handler = logging.StreamHandler(sys.stdout)
+    handler.setLevel(logging.DEBUG)
```

Comment:
Does this need to be set, as there is only one handler and level is already set for the logger?